src/mlir/verona-mlir.cc

```diff
@@ -105,8 +104,6 @@ int main(int argc, char** argv)
   // MLIR boilerplace
   mlir::registerAllDialects();
```

Comment:
I'm somewhat confused by what the deal is with the global registry now, but I don't think we need to bother doing this anymore.

Reply:
Ugh, left over from the migration. Removing...